docs/creating-ha-clusters.md

```diff
@@ -0,0 +1,84 @@
+# Creating Clusters With HA Control Plane  <!-- omit in toc -->
+Currently clusterctl supports creating clusters with a single instance controlplane. This document outlines the steps for standing up an cluster with an HA controlplane.
+
+## Contents <!-- omit in toc -->
+- [Prerequisites](#prerequisites)
+  - [Creating cluster with single instance controlplane.](#creating-cluster-with-single-instance-controlplane)
+- [Growing controlplane into HA controlplane.](#growing-controlplane-into-ha-controlplane)
+- [Troubleshooting](#troubleshooting)
+  - [Instances for the new machines not getting created](#instances-for-the-new-machines-not-getting-created)
+  - [Instances were created and initialized, but nodes don't show up on running 'kubectl get no'](#instances-were-created-and-initialized-but-nodes-dont-show-up-on-running-kubectl-get-no)
+
+## Prerequisites
+### Creating cluster with single instance controlplane.
+Following the [getting started guide](docs/getting-started.md), please setup a cluster with a single control plane instance.
```

Comment:
I think we should probably recommend that a user create the HA control plane prior to joining any worker nodes, otherwise we are risking more potential impact to running workloads if they start deploying applications prior to expanding the control plane.

This would likely require the upcoming clusterctl pivot phase which I'm currently working as part of: https://github.com/kubernetes-sigs/cluster-api/issues/554

Reply:
OK. Let's wait on this till then.

Reply:
Per offline discussion, let us get this doc out there, esp considering we want to announce the multinode controlplane. I'll keep the issue open so that we can iterate to get this where we want.